content/guides/devcontainers.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Dev Containers
+ogTitle: How to use Depot with Dev Containers
+description: Leverage Depot to build your Dev Containers on demand with our `depot configure-docker` command.
+---
+
+# Dev Containers
+
+## Why use Depot with Dev Containers?
+
+[Dev Containers](https://code.visualstudio.com/docs/devcontainers/containers) are becoming a popular way to leverage a container as a fully featured development environment directly integrated with Visual Studio Code. You can open any folder inside a container and use the full power of VS Code inside.
+
+With Depot, you can build your Dev Containers on demand with instant shared caching across your entire team.
+
+## How to use Depot with Dev Containers
+
+### Create a project
+
+This project represents the Dev Container you want to build and share across your team. You can create a project in Depot by following these steps:
+
+1. Go to your [organizations inside of Depot](/orgs)
+2. Click on the `Create Project` button
+3. Enter a project name
+4. Choose your connection type, `Depot Hosted` or [`Self-Hosted`](/docs/self-hosted/overview)
+5. Click `Create Project`
+
+### Install and configure the `depot` CLI
+
+You must install the [`depot` CLI](/docs/cli/installation) to configure your repository and Docker environment to leverage Depot builders.
+
+Once the CLI is installed, you can configure your environment:
+
+1. Run `depot login` to login to your Depot account
+2. Change into the root of your project directory
+3. Run `depot init` to link your project to your repository; this will create a `depot.json` directory in the current directory
+
+_Note: Another option is to specify the Depot project ID with the environment variable `DEPOT_PROJECT_ID` instead of running `depot init`_
```

Comment:
Do we know if you can set env vars with devcontainers, like for the build? If not, I think we should omit this.

```suggestion
```